account_invoice_validate_send_email/models/account_move.py

```diff
@@ -71,6 +71,10 @@ def action_post(self):
                 )
             ):
                 continue
+            if not move.message_follower_ids.mapped("partner_id").filtered(
```

Comment:
```suggestion
            # No message should be created when none of the non-internal followers has an email, since having the message log in the chatter is confusing (one would assume that the message has actually been delivered the followers).
            if not move.message_follower_ids.mapped("partner_id").filtered(
```